src/components/select/_select.scss

```diff
@@ -62,6 +62,10 @@
     color: $text-01; // For the options to show in IE11
   }
 
+  .#{$prefix}--select-option[disabled] {
+    opacity: 0.5;
+  }
```

Comment:
Could we also add in `cursor: not-allowed;`, for consistency?

Reply:
Done!